### ISSUE TITLE
WebAssembly/WebGL support

### ISSUE THREAD

**Issue by AderitoSilva** [Noesis/Managed#8]
Is there planned support for WebAssembly and WebGL, without Unity?

**Comment by jesusdesantos** [comment]
Yes, low-priority for now though. Please, use our forums to get more information about features and issues. Thanks!

**Comment by AderitoSilva** [comment]
> Yes, low-priority for now though. Please, use our forums to get more information about features and issues. Thanks!

That's great! Because Noesis is the only platform I'm aware of that is one-on-one with WPF, it's awesome for WPF developers that want to port to other platforms. WebAssembly is basically the only main platform Noesis still lacks. I'm keeping a close eye on Noesis and, likely, will begin investing in new projects using it, as WebAssembly support is planned for the future.

Thanks a lot!

**Comment by jesusdesantos** [comment]
WebAssembly + WebGL (and soon WebGPU) is fully supported at core level (example, https://www.noesisengine.com/xamltoy/). It is available in C++ SDK and Unity. Getting this in pure C# is just setting up the compiler tools to build WebAssembly (the wasm library for Noesis is already available and used by Unity).

This is a task that could be even done from the outside without gettting access to internal Noesis things.

**Comment by aienabled** [comment]
I've worked on it together with @jesusdesantos lately. Alas, no luck with .NET 7 WASM. You can find more details in my message on the forums https://www.noesisengine.com/forums/viewtopic.php?p=16148#p16148
